README.md

```diff
@@ -70,8 +70,8 @@ This application requires a properties file at `./go-update.properties` that con
 
 ```
 db.host=${hostname}
-db.user=${root_username}
-db.password=${root_password}
+db.user=${username}
+db.password=${password}
```

Comment:
When we incorporate this step into Jenkins, we'll want to make these properties have the same convention as the other release steps (https://github.com/reactome/release-jenkins-utils/blob/develop/jenkins/reactome-jenkins-release-config-template.properties). Might be worth changing these now to work with 'curator.database.*'

Reply:
The most current version of release-common-lib loads properties like `db.user`, not `curator.dabase.user`. Once release-common-lib handles different properties, those changes can be made here.
https://github.com/reactome/release-common-lib/blob/develop/src/main/java/org/reactome/release/common/ReleaseStep.java#L41-L45